db/migrate/20230717220128_create_counter_metrics.rb

```diff
@@ -0,0 +1,12 @@
+class CreateCounterMetrics < ActiveRecord::Migration[5.2]
+  def change
+    create_table :counter_metrics do |t|
+      t.string :worktype
+      t.string :resource_type
+      t.string :work_id
```

Comment:
should these all be strings or should `work_id`, `total_item_investigations` and `total_item_requests` be integers? should `date` be a date or a string?